scripts/ciTest.js

```diff
@@ -63,11 +63,11 @@ function runTests() {
     var buildTestDir = path.join(__dirname, "..", "jscomp", "build_tests");
     var linkCmd = `npm link ../..`;
     console.log(linkCmd);
-    cp.execSync(linkCmd, {
-      cwd: buildTestDir,
-      stdio: [0, 1, 2],
-      encoding: "utf8",
-    });
+    // cp.execSync(linkCmd, {
+    //   cwd: buildTestDir,
+    //   stdio: [0, 1, 2],
+    //   encoding: "utf8",
+    // });
```

Comment:
Was this change committed accidentally?

Reply:
Oops, sorry, didn't notice it. Nice catch, I'll remove it.